clang-version.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Different License?

Reply:
Yes because this is from the Linux kernel's scripts folder.